tests/lcs/core/debugging/log.livecodescript

```diff
@@ -0,0 +1,55 @@
+﻿script "log"
+/*
+Copyright (C) 2019 LiveCode Ltd.
+
+This file is part of LiveCode.
+
+LiveCode is free software; you can redistribute it and/or modify it under
+the terms of the GNU General Public License v3 as published by the Free
+Software Foundation.
+
+LiveCode is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or
+FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received a copy of the GNU General Public License
+along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
+
+on TestLog
+   local tStackToRun
+   put the effective filename of me into tStackToRun
+   set the itemdelimiter to slash
+   if the environment is "server" then
+      put "_log.lc" into item -1 of tStackToRun
+   else
+      put "_log.livecodescript" into item -1 of tStackToRun
+   end if
+
+   local tOptions
+   if the environment contains "command line" then
+      put "-ui" into tOptions
+   end if
+   set the itemdelimiter to comma
+
+   local tArgs, tResult
+   repeat for each item tLogMessage in "logPublic,logPrivate"
+      put tLogMessage && "true" into tArgs
+      TestRunStack tOptions, tStackToRun, tArgs
+      if the result is not empty then
+         TestDiagnostic the result
+      else
+         TestDiagnostic it
+      end if
+      TestAssert "log message" && tLogMessage, word 1 to -1 of line 1 of it is true
```

Comment:
@runrevmark are you sure there's an issue on server? We dump the log on failure. I think there might be an issue with the test not handling the mix of stdout and stderr we get in shell here. I'm thinking that rather than testing for true we generate a uuid and check it's somewhere in the output.

Reply:
Well there's definitely a difference between what is emitted when running the tests in the standalone engine compared to the server engine (the log tests fail in the standalone test phase, but doesn't end up meaning all the individual 'ok' lines get dumped to stdout/stderr at the end, as far as I can see at least...)

Reply:
I don't think it's running under the server engine is it? I can't access more than the normal output for standalone test phase then the dumped log

Reply:
Hah - sorry - you are right - not sure where I thought I saw server ;) So, yes - this is just because the log tests are failing :D